kafka/pykafka/topic.py

```diff
@@ -52,12 +58,13 @@ def fetch_offsets(self, offsets_before, max_offsets=1):
             output.update(res.topics[self.name])
         return output
 
-    def get_producer(self):
-        return Producer(self)
+    def earliest_available_offset(self):
+        """Get the earliest offset for each partition of this topic."""
+        return self.fetch_offset_limits(OffsetType.EARLIEST)
 
-    def latest_offsets(self):
+    def latest_available_offset(self):
         """Get the latest offset for each partition of this topic."""
-        return self.fetch_offsets(OFFSET_LATEST)
+        return self.fetch_offset_limits(OffsetType.LATEST)
```

Comment:
Should be `earliest_available_offsets` and `latest_available_offsets` since it returns multiple results.